components/style/selector_matching.rs

```diff
@@ -165,39 +165,35 @@ impl Stylist {
         // Take apart the StyleRule into individual Rules and insert
         // them into the SelectorMap of that priority.
         macro_rules! append(
-            ($style_rule: ident, $priority: ident, $importance: expr, $count: expr) => {
```

Comment:
Now that I think about it, with this change the two `SelectorMap`s in a given `PerOriginSelectorMap` become identical… so we don’t need to have two anymore.

I’ll do that in a new commit and add a comment here that we insert empty blocks in case CSSOM adds something to them later.